Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json

```diff
@@ -2,5 +2,5 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"Agent Health",
     "author": "Microsoft",
-    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 200 }]
+    "galleries": [{ "type": "workbook", "resourceType": "", "order": 200 }]
```

Comment:
for this part we can just have:

`"galleries": []`

Also let's add a description here saying something like:

"description": "not shown in any gallery, this workbook is deprecated, please use Workbooks/Azure Monitor - Agents/Overview"